### DISCUSSION TITLE
Welcome to course-odcm Discussions!

### DISCUSSION THREAD

**Discussion by hannesdatta** [hannesdatta/course-odcm#33] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Akram-Coding** [comment]
Hello everyone,

My name is Akram Benmrit. I am currently enrolled in the masters program Marketing Analytics and work as a Data Analist. My goals is to discover all the possibilities of data and to make it a bridge instead of a wall. I also contributed to the course repository and made a pull request.

I also would like to ask @hannesdatta  to add the slides.html file of week 3 to the repository. I believe that is the issue with the 404 error which shows in docs/modules/week3/.

![image](https://github.com/hannesdatta/course-odcm/assets/143732554/3e3b2fd2-6942-4770-a687-4e8d80708509)
